simplq/src/main/java/me/simplq/controller/TokenController.java

```diff
@@ -17,13 +11,24 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import me.simplq.controller.model.token.CreateTokenRequest;
+import me.simplq.controller.model.token.MyTokensResponse;
+import me.simplq.controller.model.token.TokenDeleteResponse;
+import me.simplq.controller.model.token.TokenDetailResponse;
+import me.simplq.controller.model.token.TokenNotifyResponse;
+import me.simplq.service.TokenService;
+
 @RestController
 @RequestMapping("/v1")
-@RequiredArgsConstructor
 public class TokenController {
 
   private final TokenService tokenService;
 
+  @Autowired
+  public TokenController(TokenService tokenService) {
+    this.tokenService = tokenService;
+  }
+
```

Comment:
Why did we need to do this?

Reply:
Well, if we look at SOLID principle, specifically at dependecy inversion, we need to provide all dependecies via constructor. Also in tests we can mock dependencies and provide them easily via constructor.

Reply:
Lombok just generates the same constructor for us, without having to write this redundant code. And @Autowired on constructors are optional, you can skip it in spring.